src/main/java/com/yugabyte/sample/common/CmdLineOpts.java

```diff
@@ -540,6 +544,7 @@ public static CmdLineOpts createFromArgs(String[] args) throws Exception {
     options.addOption("yql_password", true,
         "Use authentication with the YQL client using the provided password. " +
             "If this option is set, yql_username option should be used too.");
+    options.addOption("concurrent_clients", true, "How many client connections to establish.");
```

Comment:
Maybe the option describe should say something like "The number of client connections to establish to each host in the YugaByte DB cluster."?